Funda mapjes - kopie/styles/nieuwefundastyle.css

```diff
@@ -0,0 +1,342 @@
+:root{
+    --primaire-color: #F7A100;
+    --secundaire-color: #0071B3;
+    --primaire-lettertype: Proxima Nova;
+    --navbarlettertype: 15px;
+}
+
+.maandlasten {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: proxima nova, sans-serif;
+}
+
+/* BEGIN HEADER */
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    /* width: 414px; */
+    height: 60px;
+    border: 0px;
+    top: 0px;    
+    background-color: var(--primaire-color);
+}
+
+.navbar-left {
+    display: flex;
+    justify-content: space-between;
+}
+
+.button-menu {
+    height: 100% ;
+    background-color: var(--primaire-color);
+    border-color: var(--primaire-color);
+    border: 0px;
+    color: white;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    font-size: 12px;
```

Comment:
Ik zou geen gebruik maken van `px` voor de font-size tenzij dit op bepaalde plekken een bewuste keuze is, zodat de tekst kan meeschalen wanneer de gebruiker het zoom niveau aanpast.